kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverNames.kt

```diff
@@ -13,175 +19,268 @@ import org.gradle.api.tasks.TaskContainer
+    public fun koverVerifyName(buildVariant: String): String {
         return verifyTaskName(buildVariant)
     }
 
     /**
      * Name of the coverage logging task for [buildVariant] Android build variant for Android projects.
      */
-    public fun androidLog(buildVariant: String): String {
+    public fun koverLogName(buildVariant: String): String {
         return logTaskName(buildVariant)
     }
+
+    // ===
+    // Deprecations
+    // Remove in 0.9.0
+
+    /**
+     * ID of Kover Gradle Plugin.
+     */
+    public const val PLUGIN_ID = "org.jetbrains.kotlinx.kover"
+
+    /**
+     * Name of the configuration to add dependency on Kover reports from another project.
+     */
+    public const val DEPENDENCY_CONFIGURATION_NAME = "kover"
+
+    /**
+     * Name of the project extension to configure Kover measurements.
+     */
+    public const val PROJECT_EXTENSION_NAME = "kover"
+
+    /**
+     * Name of the project extension to configure Kover reports.
+     */
+    public const val REPORT_EXTENSION_NAME = "koverReport"
+
+    @Deprecated(
+        message = "Constant was renamed. Please refer to migration guide in order to migrate: ${KoverMigrations.MIGRATION_0_7_TO_0_8}",
+        level = DeprecationLevel.ERROR
+    )
+    public const val DEFAULT_XML_REPORT_NAME = "koverXmlReport"
+
+    /**
+     * Name of the binary report generation task for Kotlin JVM and Kotlin multiplatform projects.
+     */
+    public const val DEFAULT_BINARY_REPORT_NAME = "koverBinaryReport"
+
+    /**
+     * Name of the HTML report generation task for Kotlin JVM and Kotlin multiplatform projects.
+     */
+    public const val DEFAULT_HTML_REPORT_NAME = "koverHtmlReport"
+
+    /**
+     * Name of the verification task for Kotlin JVM and Kotlin multiplatform projects.
+     */
+    public const val DEFAULT_VERIFY_REPORT_NAME = "koverVerify"
+
+    /**
+     * Name of the coverage logging task for Kotlin JVM and Kotlin multiplatform projects.
+     */
+    public const val DEFAULT_LOG_REPORT_NAME = "koverLog"
 }
 
-/**
- * Name of the XML report generation task for Kotlin JVM and Kotlin multiplatform projects.
- *
- * Has the same value as [KoverNames.DEFAULT_XML_REPORT_NAME].
- */
-public val TaskContainer.koverXmlReportName
-    get() = KoverNames.DEFAULT_XML_REPORT_NAME
 
-/**
- * Name of the HTML report generation task for Kotlin JVM and Kotlin multiplatform projects.
- *
- * Has the same value as [KoverNames.DEFAULT_HTML_REPORT_NAME].
- */
-public val TaskContainer.koverHtmlReportName
-    get() = KoverNames.DEFAULT_HTML_REPORT_NAME
+// ===
+// Deprecations
+// Remove in 0.9.0
 
-/**
- * Name of the binary report generation task for Kotlin JVM and Kotlin multiplatform projects.
- *
- * Has the same value as [KoverNames.DEFAULT_BINARY_REPORT_NAME].
- */
-public val TaskContainer.koverBinaryReportName
-    get() = KoverNames.DEFAULT_BINARY_REPORT_NAME
+@Deprecated(
+    message = "...",
```

Comment:
Do you plan to provide a proper message?

Reply:
Yes, now there are no correct deprecations